src/test/osd/CMakeLists.txt

```diff
@@ -74,6 +76,10 @@ target_link_libraries(unittest_hitset osd global ${BLKID_LIBRARIES})
 add_executable(unittest_osd_osdcap
   osdcap.cc
 )
+if(HAS_VTA)
+  set_source_files_properties(osdcap.cc PROPERTIES
+    COMPILE_FLAGS -fno-var-tracking-assignments)
+endif()
```

Comment:
ok thanks, i didn't see the commit message. was that warning already present, or somehow introduced by moving unit.h -> unit.cc?